base/mmap.jl

```diff
@@ -282,6 +282,17 @@ const MS_ASYNC = 1
 const MS_INVALIDATE = 2
 const MS_SYNC = 4
 
+"""
+    Mmap.sync!(m::Array, flags::Integer=MS_SYNC)
+
+Forces synchronization between the in-memory version of a memory-mapped `Array` or
+`BitArray` and the on-disk version. The optional `flags` argument
+is used on UNIX to control the behavior of `msync`. There are three options:
```

Comment:
the all caps version is a trademark, not an acronym, right? we should just be consistent about it